### PR TITLE
docs(sentry-cli): Fix "deploys new" command

### PR DESCRIPTION
**Pull request by MIrinkov**

### CODE REVIEW

**Review by armenzg — APPROVED** [review]
Thank you, @MIrinkov !
I noticed this discrepancy last time I played with this but I forgot about updating it.

Does the command work well these days?

**Comment by armenzg** [comment]
And this is the associated page for this PR:
https://docs.sentry.io/product/releases/setup/#using-release-automation

File: src/docs/product/releases/setup/index.mdx [hunk 1]
```diff
@@ -25,7 +25,7 @@ Notifying Sentry of a release enables auto discovery of which commits to associa
 If you're using [Release Automation](/product/releases/setup/release-automation/), let Sentry know you’ve deployed by sending an additional request after creating a release:
 
 ```bash
-sentry-cli releases --org ORGANIZATION_SLUG deploys new VERSION -e ENVIRONMENT
+sentry-cli releases --org ORGANIZATION_SLUG deploys VERSION new -e ENVIRONMENT
```
[the remark this answers is not in the document]
Comment (MIrinkov) [reply src/docs/product/releases/setup/index.mdx:28]: The command works great.
I've copy pasted the wrong version into my console, got a very descriptive error message back, and then realized there's a "typo" in the docs.
Thanks to "edit this page" link in the bottom, submitting a fix was quick and painless.
Thanks for reviewing :)

[the remark this answers is not in the document]
Comment (armenzg) [reply src/docs/product/releases/setup/index.mdx:28]: My pleasure. Thanks for your help!

**Pull request merged**